ci-operator/config/opendatahub-io/model-registry/opendatahub-io-model-registry-main.yaml

```diff
@@ -21,7 +21,8 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: e2e-odh-cfo
+- as: e2e-odh-cfo-cron
```

Comment:
I think that `cfo` means Cloudflare Operator here :) We should use `mro` for Model Registry Operator.